cmd/create/cluster/cmd.go

```diff
@@ -2001,7 +2001,7 @@ func run(cmd *cobra.Command, _ []string) {
 				!serviceNetwork.Contains(subnetIP) {
 				subnets = append(subnets, subnet)
 			} else {
-				excludedSubnetsDueToCidr = append(excludedSubnetsDueToCidr, awssdk.StringValue(subnet.SubnetId))
+				excludedSubnetsDueToCidr = append(excludedSubnetsDueToCidr, awssdk.ToString(subnet.SubnetId))
```

Comment:
Returning a slice of subnets from `getInitialValidSubnets` will make this conversion redundant.

Reply:
When changing it to a slice it still requires it to use `ToString`